src/multi_vector_simulator/F2_autoreport.py

```diff
@@ -614,7 +614,7 @@ def create_app(results_json, path_sim_output=None):
     marker = staticmap.CircleMarker(coords, "#13074f", 15)
     map_static.add_marker(marker)
     map_image = map_static.render(zoom=14)
-    map_image.save(os.path.join(REPORT_PATH, "assets", "proj_map_static.png"))
+    map_image.save(os.path.join(asset_folder, "proj_map_static.png"))
```

Comment:
Is it good practice to still have `"proj_map_static.png"` as a string? Used sufficiently few times?

Reply:
it is only generated in F2, so used twice. We could pass over the F2 to sanitze it but I need time to do so and I place it in the not so important not urgent category of my Heisenberg chart :)